cartridges/openshift-origin-cartridge-python/usr/versions/2.7-scl/lib/update-configuration

```diff
@@ -8,19 +8,20 @@ function update-configuration {
     # SCL installs of python 2.7
     sclpath=$(dirname $(scl enable python27 "which python"))
     env_dir="${OPENSHIFT_PYTHON_DIR}/env"
```

Comment:
It is only a local variable. It dies out soon enough that I felt that it does not matter.